woocommerce/woocommerce-android
# Add toolbar for ShippingPackageSelectorFragment

Part of #10570 

### Description

This PR introduces individual toolbars for ShippingPackageSelectorFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA -> Ship from -> Ship to -> Package details -> Package selected) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="243" alt="Screenshot 2024-02-12 at 6 43 37 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/6398f25b-9b29-4994-a94e-2c07bd3b0b6a">
